addon/components/collapsible-list.js

```diff
@@ -67,6 +67,7 @@ export default Component.extend({
       isExpanded: true,
       'panelWrapper.style.display': null,
     });
+    this.triggerEvent('onAfterShow', item);
```

Comment:
It looks like adding the `onAfterShow` callback to `simpleShow()` will cause the callbacks to execute in an inconsistent order: `onAfterShow` executes first, then `onShow` after.

IMO the `onAfterShow` callback makes sense only for the animated version because for the non-animated version both `onShow` and `onAfterShow` would execute one after another without any delay. So I'd say let's remove it from `simpleShow()` and possibly rename it something like `onShowComplete` so this way it's a bit more clear that "showing" takes some amount of time before it completes. We then just need to add to the docs that `onShowComplete` is relevant for animated collapsibles/accordions.

Reply:
So you're right, but I still think it makes sense to have both events for both the animated/simple show.
Imagine that you have an app which has a no/low animation mode (e.g.: iOS has the same mode as animations give some people motion sickness). If in the animated mode you do something `onShow` and then `onAfterShow`, the you need to treat the two modes differently.

I added a test to make sure the two events are called in the right order. Let me know what you think.

Reply:
Okay, yeah that makes sense - let's keep them for both cases.